webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 const fs = require('fs')
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const TerserPlugin = require('terser-webpack-plugin')
```

Comment:
This is great because UglifyJsPlugin appears to now how a vulnerability  (https://npmjs.com/advisories/1426)